### PR TITLE
Wei Arcade Game

### PR DESCRIPTION
**Pull request by Weeeeeeeeei**
<h2>Summary</h2>
 
Lady black swan have to survive under the attack, try to not bump into something that you should not bump in...

<h2>Details</h2>

The player controls the swan to eat the tomatoes, and each tomato eaten can increase the size, but the there are others can reduce the body or kill you.

<h2>Visuals</h2>
.....
<h2>How to Test</h2>
Just follow the instruction and play it.

- How do we play this?
Mouse clicks and WA to control the direction.

- What known issues are there?
If your body length is 0, and got hit by the reduce bullet, the game will stuck.

<h2>How to Review</h2>
Say anything that you want.

- What do you want feedback on?
Anything

### CODE REVIEW

**Comment by ekaterina392** [comment]
Hi Wei,

I figured out only how to control the snake but I did not get any collision with the white circles. Maybe you are planning to make them as obstacles in the future?
I think you should change a little the movements of the snake, probably make them smoother and faster, because it is not obvious that the snake can move at the first glance.

File: Assets/Wei Wu/scripts/AppleController.cs [hunk 1]
```diff
@@ -0,0 +1,53 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class AppleController : MonoBehaviour
```
Comment (jasonzli) [inline Assets/Wei Wu/scripts/AppleController.cs:5]: namespace

File: Assets/Wei Wu/scripts/CameraFollow.cs [hunk 2]
```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class CameraFollow : MonoBehaviour
```
Comment (jasonzli) [inline Assets/Wei Wu/scripts/CameraFollow.cs:5]: namespaces

File: Assets/Wei Wu/scripts/FirePoint.cs [hunk 3]
```diff
@@ -0,0 +1,66 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class FirePoint : MonoBehaviour
```
Comment (jasonzli) [inline Assets/Wei Wu/scripts/FirePoint.cs:5]: namespace

File: Assets/Wei Wu/scripts/SnakeController.cs [hunk 4]
```diff
@@ -0,0 +1,93 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class SnakeController : MonoBehaviour
```
Comment (jasonzli) [inline Assets/Wei Wu/scripts/SnakeController.cs:5]: namespace

**Review by jasonzli — CHANGES_REQUESTED** [review]
Exciting start! Needs namespaces but you've organized things well. Consider a more specific scene name.

**Comment by Hectorgamecreator** [comment]
The game art is amazing; I loved the story plot behind the snake game. I would like to see if you have a game title for your loading screen. Overall, I had a fun time trying to collect as many eggs as I could.  Also, what is your game's screen size for the game?

**Pull request closed without merging**